src/dlangui/dialogs/settingsdialog.d

```diff
@@ -256,7 +256,7 @@ class ExecutableFileNameEditItem : SettingsItem {
         import dlangui.dialogs.filedlg;
         TextWidget lbl = new TextWidget(_id ~ "-label", _label);
         FileNameEditLine ed = new FileNameEditLine(_id ~ "-filename-edit");
-        ed.addFilter(FileFilterEntry(UIString.fromRaw("MESSAGE_EXECUTABLES"d), "*.exe", true));
+        ed.addFilter(FileFilterEntry(UIString.fromRaw("MESSAGE_EXECUTABLES"c), "*.exe", true));
```

Comment:
`UIString` has two functions `fromRaw()` and `fromId()`. If you want use id like `MESSAGE_EXECUTABLES` you should use `fromId()`. `UIString.fromRaw()` just shows "MESSAGE_EXECUTABLES" not "Executable files" when English is chosen.